python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -89,6 +89,13 @@ class PiffPsfDeterminerConfig(BasePsfDeterminerTask.ConfigClass):
         ),
         default="scipy",
     )
+    piffPixelGridFitCenter = pexConfig.Field[bool](
+        doc="PixelGrid can re-estimate center if this option is True. "
+        "Will use provided centroid if set to False. Default in Piff "
+        "is set to True. Depends on how input centroids can be trust. "
+        "Ignore if piffPsfConfigYaml is not None.",
+        default=True
```

Comment:
I think leaving the default to `True` makes sense for now, when we're focused on DP1 and yet don't want to break (e.g.) HSC, but we should make sure we come back to this and try to make it `False` by default.